.jenkins/caffe2/test.sh

```diff
@@ -134,26 +134,24 @@ if [[ $BUILD_ENVIRONMENT == *-rocm* ]]; then
   rocm_ignore_test+=("--ignore $caffe2_pypath/python/ideep/pool_op_test.py")
 fi
 
-# NB: Warnings are disabled because they make it harder to see what
-# the actual erroring test is
 echo "Running Python tests.."
-if [[ "$BUILD_ENVIRONMENT" == *py3* ]]; then
```

Comment:
Is it removed because all CIs are on py3?

Reply:
Yes. Updated PR description.